LLDBDebugger/LLDBPlugin.h

```diff
@@ -33,6 +33,7 @@
 #include "LLDBProtocol/LLDBConnector.h"
 #include "LLDBProtocol/LLDBEvent.h"
 #include "LLDBProtocol/LLDBRemoteConnectReturnObject.h"
+#include "../Plugin/clDebuggerTerminal.h"
```

Comment:
I think we can simply include `#include <clDebuggerTerminal.h>` there is no need for the relative path here

Reply:
Cool. Not sure why I did that!